# Request
# The namespace of the robot for which the state is required
string name_space
nav_msgs/Path path
---
#Response
string SUCCESS = "success"
string UNKNOWN_ERROR ="unknown_error"
string ROBOT_NOT_FOUND = "robot_not_found"

# status code of request
string statuscode "unknown error"